src/gc/windows/gcenv.windows.cpp

```diff
@@ -344,6 +343,501 @@ static size_t GetRestrictedPhysicalMemoryLimit()
     return g_RestrictedPhysicalMemoryLimit;
 }
 
+// This function checks to see if GetLogicalProcessorInformation API is supported. 
```

Comment:
`GetLogicalProcessorInformation` is supported since Windows Server 2003 according to https://msdn.microsoft.com/en-us/library/windows/desktop/ms683194(v=vs.85).aspx. We should not need to have the fallback path in standalone GC - the fallback is leftover from time when we needed to support old OSes

Reply:
`GetLogicalProcessorInformation` had a bug in Windows Server 2003. It should not be a conven since Windows Server 2003 is not supported for nether .NET Core nor .NET Framework.